### PR TITLE
RT 3200 and v0.40.1 client upgrades

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Moonriver Runtime upgrade to 3200 and v0.40.1 client upgrades.

### Checklist

- [x] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by albertov19 — DISMISSED** [review]

File: variables.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 networks:
   development:
-    build_tag: v0.40.0
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.40.0-3200-latest
+    build_tag: v0.40.1
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.40.1-3200-ae11
```
Comment (eshaben) [inline variables.yml:4]: we've been using latest, can we use that instead?

Comment (eshaben) [reply variables.yml:4]: [v0.40.1-3200-latest](https://hub.docker.com/layers/moonbeamfoundation/moonbeam-tracing/v0.40.1-3200-latest/images/sha256-f69b36f53b9b173fca333aa1d822101761d9bff6275aa1cd6af03fd96d62a036?context=explore)

**Review by eshaben — CHANGES_REQUESTED** [review]
Thanks!

File: variables.yml (hunk 1 above)
Comment (albertov19) [reply variables.yml:4]: I prefer using the official tag being used by the release comms

Comment (eshaben) [reply variables.yml:4]: Haha, we change our mind on this like every 6 months 😂

**Review by eshaben — APPROVED** [review]

**Pull request merged**